python/ray/train/lightning/_lightning_utils.py

```diff
@@ -31,10 +31,10 @@ def import_lightning():  # noqa: F402
 _TORCH_GREATER_EQUAL_1_12 = Version(torch.__version__) >= Version("1.12.0")
 _TORCH_FSDP_AVAILABLE = _TORCH_GREATER_EQUAL_1_12 and torch.distributed.is_available()
 
-if _LIGHTNING_GREATER_EQUAL_2_0:
+try:
     from lightning.pytorch.plugins.environments import LightningEnvironment
     from lightning.pytorch.strategies import FSDPStrategy
-else:
+except ModuleNotFoundError:
     from pytorch_lightning.plugins.environments import LightningEnvironment
     from pytorch_lightning.strategies import DDPFullyShardedStrategy as FSDPStrategy
```

Comment:
I remember there was a reason we didn't do this, but would it be possible to figure out how to do something like:
```python
    from pl.plugins.environments import LightningEnvironment
    from pl.strategies import FSDPStrategy
```

Reply:
The reason here is we cannot do 
```python
import lightning.pytorch as pl
pl.plugins.environments.LightningEnvironment

# which is similar as
import ray
ray.train.torch.TorchTrainer
```

That's why we explicitly import `LightningEnvironment` here.


```
from pl.plugins.environments import LightningEnvironment
from pl.strategies import FSDPStrategy
```
I remember I searched for quite a while but the python syntax doesn't support using an alias as the prefix of import path.

Reply:
I remember I searched for quite a while, but finally found that the python syntax doesn't support using an alias as the prefix of import path.

```
from pl.plugins.environments import LightningEnvironment
from pl.strategies import FSDPStrategy
```

Reply:
Oh that seems a little weird, let me take a look at this today

Reply:
I think we can just do this? 

```python
LightningEnvironment = pl.plugins.environments.LightningEnvironment

if _LIGHTNING_GREATER_EQUAL_2_0:
    FSDPStrategy = pl.strategies.FSDPStrategy
else:
    FSDPStrategy = pl.strategies.DDPFullyShardedStrategy
```

Reply:
The FSDP import should be good, but calling `pl.plugins.environments.LightningEnvironment` will raise an error. We can only use `import ... from `clause to use `LightningEnvironment`.

Reply:
What about we do

```python

try:
    from lightning.pytorch.plugins.environments import LightningEnvironment
except:
    from pytorch_lightning.plugins.environments import LightningEnvironment

if _LIGHTNING_GREATER_EQUAL_2_0:
    FSDPStrategy = pl.strategies.FSDPStrategy
else:
    FSDPStrategy = pl.strategies.DDPFullyShardedStrategy
```

Reply:
Ah, yeah I guess that seems fine/clean enough, thanks for lookign into this.